core/src/Kokkos_ExecPolicy.hpp

```diff
@@ -905,6 +924,17 @@ require(const PolicyType p, WorkItemProperty::ImplWorkItemProperty<P>) {
   return typename Impl::PolicyPropertyAdaptor<
       WorkItemProperty::ImplWorkItemProperty<P>, PolicyType>::policy_out_t(p);
 }
+
+template <typename Policy>
+/*constexpr*/ typename Impl::PolicyPropertyAdaptor<DesiredOccupancy,
+                                                   Policy>::policy_out_t
+prefer(Policy const& p, DesiredOccupancy o) {
```

Comment:
Avoid using `o` as a name variable, depending on the font it might look like zero.